app/assets/javascripts/components/components/load_more.jsx

```diff
@@ -1,9 +1,9 @@
 import { FormattedMessage } from 'react-intl';
 
 const LoadMore = ({ onClick }) => (
-  <a href='#' className='load-more' onClick={onClick}>
+  <span tabIndex='0' className='load-more' role='button' onClick={onClick}>
```

Comment:
I'm not sure about this one, I haven't run it since making this change. I can run this evening and double check.